### DISCUSSION TITLE
Would it be possible to package it in some format to distribute Flet pages in a Flutter app, rather than having to run a python server?

### DISCUSSION THREAD

**Discussion by moluopro** [flet-dev/flet#1169] [General]
Would it be possible to package it in some format to distribute Flet pages in a Flutter app, rather than having to run a python server? And I would like to be able to distribute Flet as a Flutter component instead of a page, I wonder if that is possible and how the Flet page would interact with the Flutter application and pass data?

**Comment by FeodorFitsner** [comment]
You mean distributing it as a mobile app?
